packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -389,6 +389,107 @@ <h2>
                 </div>
             </div>
         </oui-step-form>
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_training_jobs_submit_select_ssh' | translate }}"
+            data-editable="!$ctrl.isSubmit"
+        >
+            <oui-message type="info" dismissable="true">
+                <span
+                    class="d-block"
+                    data-translate="pci_projects_project_training_jobs_submit_select_ssh_tips_1"
+                ></span>
+            </oui-message>
+            <oui-field
+                data-size="xl"
+                data-label="{{:: 'pci_projects_project_training_jobs_submit_select_ssh_default_select' | translate  }}"
+            >
+                <oui-switch
+                    data-model="$ctrl.enabledSshPublicKey"
+                    name="enabledSshPublicKey"
+                    on-change="$ctrl.changeEnabledSshPublicKey(modelValue)"
+                >
+                </oui-switch>
+            </oui-field>
+            <div
+                class="oui-field__control"
+                data-ng-if="$ctrl.enabledSshPublicKey"
+            >
+                <div class="oui-inline-adder mb-2">
+                    <div
+                        class="oui-inline-adder__form"
+                        data-ng-repeat="sshKey in $ctrl.addedSshKeys track by $index"
+                    >
+                        <fieldset class="oui-inline-adder__fieldset">
+                            <legend
+                                data-ng-bind=":: 'pci_projects_project_training_jobs_submit_select_ssh_legend' | translate"
+                            ></legend>
+                            <div class="oui-inline-adder__row">
+                                <div class="oui-inline-adder__field">
+                                    <oui-field
+                                        data-size="m"
+                                        data-label="{{:: 'pci_projects_project_training_jobs_submit_select_ssh_name_tips' | translate  }}"
+                                        data-ng-if="$ctrl.allKeyNames.length > 0"
+                                    >
+                                        <oui-select
+                                            id="savedKeyName"
```

Comment:
**issue(blocking):** Due to the ngRepeat (L420), this will ended up with some duplicate IDs.